proteus/mprans/RANS2P.py

```diff
@@ -1095,7 +1095,7 @@ def __init__(self,
         self.q[('f', 0)] = numpy.zeros((self.mesh.nElements_global, self.nQuadraturePoints_element, self.nSpace_global), 'd')
         self.q[('velocity', 0)] = numpy.zeros((self.mesh.nElements_global, self.nQuadraturePoints_element, self.nSpace_global), 'd')
         self.q['velocity_solid'] = numpy.zeros((self.mesh.nElements_global, self.nQuadraturePoints_element, self.nSpace_global), 'd')
-        self.q['phi_solid'] = numpy.zeros((self.mesh.nElements_global, self.nQuadraturePoints_element), 'd')
+        self.q['phi_solid'] = 10000*numpy.ones((self.mesh.nElements_global, self.nQuadraturePoints_element), 'd')
```

Comment:
I think it'd be better to define some sort of global constant INIT_PHI_SOLID=10000 as that initial scaling constant is showing up a number of times.

Reply:
Agreed. This had just been numpy.ones, which I thought was clear enough, but because that can be close enough to zero in some domains to put it in the transition region of the smoothed Heaviside function, I find it safer to make it large and positive (i.e. safely in the fluid phase).

Reply:
added in last commit